refarch-integrations/refarch-s3-integration/refarch-s3-integration-rest/refarch-s3-integration-rest-service/src/main/java/de/muenchen/refarch/integration/s3/configuration/nfcconverter/NfcRequest.java

```diff
@@ -54,7 +52,7 @@ private void convert() {
     @Override
     public Cookie[] getCookies() {
         convert();
-        return this.cookies;
+        return Arrays.copyOf(cookies, cookies.length);
```

Comment:
Just for clarification. This copy here does not guarantee that the content is not altered. Is this required in this place? If so we should refactor this to a `List<Cookie>` and use `unmodifiableList` for creation.

Reply:
Yes think that is a spotbugs fix, that slid in.
Could also remove it for now.